usage.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: usage
+title: Quarkus Usage Analytics
+subtitle: 
+permalink: /usage/
+---
+
+Starting from Quarkus 3.2 when you run Quarkus the first time in dev mode (`quarkus dev`, `mvn quarkus:dev`, etc.) you get asked if you agree to contribute anonymous build data to the Quarkus community.
```

Comment:
Starting from Quarkus 3.2, when you run Quarkus for the first time in dev mode (`quarkus dev`, `mvn quarkus:dev`, etc.), you are asked if you agree to contribute anonymous build data to the Quarkus community.